TestProjects/VisualEffectGraph_URP/ProjectSettings/ProjectSettings.asset

```diff
@@ -576,19 +576,19 @@ PlayerSettings:
   webGLThreadsSupport: 0
   webGLDecompressionFallback: 0
   scriptingDefineSymbols:
-    1: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING;URP_ENABLE_DEBUG_DISPLAY
```

Comment:
FYI - `URP_ENABLE_DEBUG_DISPLAY` is not used anymore so removed it here as well.